var/spack/repos/builtin/packages/mpifileutils/package.py

```diff
@@ -38,13 +38,14 @@ class Mpifileutils(AutotoolsPackage):
     homepage = "https://github.com/hpc/mpifileutils"
     url      = "https://github.com/hpc/mpifileutils/releases/download/v0.6/mpifileutils-0.6.tar.gz"
 
+    version('master', git='https://github.com/hpc/mpifileutils.git', branch='master')
     version('0.7', 'c081f7f72c4521dddccdcf9e087c5a2b')
     version('0.6', '620bcc4966907481f1b1a965b28fc9bf')
 
     depends_on('mpi')
     depends_on('libcircle')
     depends_on('lwgrp')
-    depends_on('dtcmp')
+    depends_on('dtcmp@1.0.3')
```

Comment:
Presumably older versions work as well, just not newer versions? If so, can you switch it to `:1.0.3`? Might also want to add a comment directly to the file so we can check to see if this version requirement ever gets fixed.